frontend/src/pages/ArtSubmission/ArtSubmission.jsx

```diff
@@ -82,6 +84,35 @@ export default function ArtSubmission() {
     updateTagsForm(newTags);
   }
 
+  useEffect(() => {
+    var options = {
+      enableHighAccuracy: true,
+      timeout: 5000,
+      maximumAge: 10000
+    };
+
+    if (!navigator.geolocation) {
+      return
+      // Geolocation is not supported by this browser.
+    }
+    navigator.geolocation.getCurrentPosition(
+      (position) => {
+        setLocation([position?.coords.longitude, position?.coords.latitude])
+      },
+      (error) => {
+        return
+      },
+      options);
+  }, [])
+  
+  if (location === [0, 0]) {
+    return (
+      <div>
+        <Spinner absCenter={true} />
+        <h1>Infinite loading? Make sure your location is enabled.</h1>
+      </div>);
+  }
```

Comment:
Maybe we use the `<ConnectionErrorMessage>` component for this.

Reply:
Just made the edit. Good suggestion too. I never saw the error, but I imagine that the h1 would look laughably large.